app/models/course/virtual_classroom.rb

```diff
@@ -4,6 +4,7 @@ class Course::VirtualClassroom < ActiveRecord::Base
   before_validation :convert_duration_to_end_at
   acts_as_readable on: :updated_at
   has_many_attachments on: :content
+  belongs_to :instructor, class_name: 'User', foreign_key: :instructor_id, inverse_of: nil
```

Comment:
Actually we have a creator_id within the table, do you want to just use that? It seems like there is no special place to allow the user to change this instructor.